helpers/deproxy.py

```diff
@@ -751,7 +758,9 @@ def run_start(self):
         tf_cfg.dbg(3, "\tStarting deproxy client")
 
         tf_cfg.dbg(4, "\tDeproxy: Client: Connect to %s:%d." % (self.conn_addr, self.port))
-        self.create_socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.create_socket(
+            socket.AF_INET if self.socket_family == "ipv4" else socket.AF_INET6, socket.SOCK_STREAM
```

Comment:
may be add here more strong check? if self.socket_family == "ipv4": socket.AF_INET else if self.socket_family == "ipv6" else raise Exception?